packages/engine/Source/Scene/createGooglePhotorealistic3DTileset.js

```diff
@@ -17,7 +17,9 @@ import Resource from "../Core/Resource.js";
  * @see GoogleMaps
  *
  * @example
- * const viewer = new Cesium.Viewer("cesiumContainer");
+ * const viewer = new Cesium.Viewer("cesiumContainer", {
```

Comment:
Should we add an explicit note to the inline docs about the need to update the geocoder, similar to what we may put in `GoogleGeocoderService`?